indico/modules/events/layout/controllers.py

```diff
@@ -78,12 +116,29 @@ def _process(self):
         f = request.files.get('file')
```

Comment:
should use brackets since it's required. but @nop33  also changed it so i guess it doesn't make much sense to change it here, too